pkg/virtualnode/resource_manager.go

```diff
@@ -196,10 +222,14 @@ func (m *defaultResourceManager) createSDKVirtualNode(ctx context.Context, ms *a
 
 func (m *defaultResourceManager) updateSDKVirtualNode(ctx context.Context, sdkVN *appmeshsdk.VirtualNodeData, ms *appmesh.Mesh, vn *appmesh.VirtualNode, vsByKey map[types.NamespacedName]*appmesh.VirtualService) (*appmeshsdk.VirtualNodeData, error) {
 	actualSDKVNSpec := sdkVN.Spec
+	for i, j := range vsByKey {
+		m.log.Info(vn.Name + "> " + i.String() + ", " + j.Name)
```

Comment:
why do we log this and in this format? `virtual node name > [index], virtual service key`

Reply:
debugging info I left in there, will remove